lib/processors/index.js

```diff
@@ -0,0 +1,69 @@
+
+/**
+ * Dependencies
+ */
+
+var debug = require('debug')('processor');
+var parseContentType = require('content-type').parse;
+var request = require('superagent');
+
+/**
+ * User-Agent string for requests.
+ *
+ * @type {String}
+ */
+var ua = 'Mozilla/5.0 (Windows NT 6.1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/41.0.2228.0 Safari/537.36'; // jshint ignore:line
```

Comment:
You dropped the "why we do this" note too?

`// Pretend to be Chrome for UA sniffing sites`